src/components/toast/toast.tsx

```diff
@@ -11,31 +14,45 @@ const ToastContainer = styled.div`
 
 type Props = {
 	message: string;
+	type: 'default' | 'warning' | 'error' | 'success';
+	timer?: number;
 	onDismiss: () => void;
 };
 
-const Toast = ({ message, onDismiss }: Props) => (
-	<div
+const Toast = ({ message, onDismiss, type }: Props) => {
+	const { state: { theme } } = useStore();
+
+	return <div
 		style={{
-			background: 'LemonChiffon',
+			display: 'flex',
```

Comment:
let's limit the `maxLength` to `335px` (mobile max is 375 - 2 * 20px padding) ... also make sure that it's centered on mobile and takes up all the width